cob_sound/ros/src/sound.cpp

```diff
@@ -5,18 +5,22 @@
 #include <std_msgs/String.h>
 #include <std_srvs/Trigger.h>
 #include <cob_sound/SayAction.h>
-#include <cob_sound/SayText.h>
+#include <cob_sound/PlayAction.h>
+#include <cob_srvs/SetString.h>
 
-class SayAction
+class SoundAction
 {
 protected:
 
   ros::NodeHandle nh_;
-  actionlib::SimpleActionServer<cob_sound::SayAction> as_;
-  ros::ServiceServer srvServer_;
+  actionlib::SimpleActionServer<cob_sound::SayAction> as_say_;
+  actionlib::SimpleActionServer<cob_sound::PlayAction> as_play_;
+  ros::ServiceServer srvServer_say_;
+  ros::ServiceServer srvServer_play_;
   ros::ServiceServer srvServer_mute_;
   ros::ServiceServer srvServer_unmute_;
-  ros::Subscriber sub_;
+  ros::Subscriber sub_say_;
+  ros::Subscriber sub_play_;
   std::string action_name_;
```

Comment:
this is obsolete ad you coded the ActionName into the advertise directly...
...or we would need two strings - one for each action